guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -0,0 +1,69 @@
+---
+group: cloud
+title: Configure Docker
+version: 2.1
+github_link: cloud/reference/docker-config.md
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+{{site.data.var.ece}} provides a Docker environment option for those who use their local environment for develop and testing, or for automation tasks. There are three, essential parts to launching the Magento Cloud Docker environment: a Magento 2 template, Docker Compose, and Magento Cloud Tools.
+
+The [Magento Cloud Docker repository](https://github.com/magento/magento-cloud-docker){:target="\_blank"} contains build information for the following [Docker hub](https://hub.docker.com/r/magento/){:target="\_blank"} images, including the following:
+
+-  PHP: magento/magento-cloud-docker-php
+    -  PHP-CLI - version 7 and later
+    -  PHP-FPM - version 7 and later
+-  NGINX:  magento/magento-cloud-docker-nginx
+
+**Prerequisites**—Select your Magento 2 template.
```

Comment:
Is it correctly aligned in 1 line?

Reply:
You know, I forgot to add the Options for the new docker:build command. I will compartmentalise this better when I do that.